### PR TITLE
Implement classic controller tas input.

### PR DESCRIPTION
**Pull request by RachelBryk**

### CODE REVIEW

File: Source/Core/DolphinWX/TASInputDlg.cpp [hunk 1]
```diff
@@ -729,14 +892,22 @@ bool TASInputDlg::TASHasFocus()
 		return false;
 }
 
+TASInputDlg::Stick* TASInputDlg::FindStickByID(int id)
+{
+	if (id == ID_MAIN_STICK)
+		return &m_main_stick;
+	else if (id == ID_C_STICK)
+		return&m_c_stick;
```
Comment (lioncash) [inline Source/Core/DolphinWX/TASInputDlg.cpp]: space here.

File: Source/Core/DolphinWX/TASInputDlg.h [hunk 2]
```diff
@@ -92,32 +95,54 @@ class TASInputDlg : public wxDialog
...
 		wxBoxSizer* m_ext_szr;
+		wxBoxSizer* m_cc_szr;
 		wxStaticBoxSizer* m_main_stick_szr;
 		wxStaticBoxSizer* m_c_stick_szr;
+		wxStaticBoxSizer* m_cc_l_stick_szr;
+		wxStaticBoxSizer* m_cc_r_stick_szr;
+
+		enum
+		{
+			cc_l_stick_x,
```
Comment (lioncash) [inline Source/Core/DolphinWX/TASInputDlg.h]: enums are usually all uppercased (at least in our codebase anyways)

File: Source/Core/DolphinWX/TASInputDlg.cpp [hunk 3]
```diff
@@ -44,8 +45,10 @@ void TASInputDlg::CreateBaseLayout()
 {
 	for (unsigned int i = 0; i < 10; ++i)
 		m_controls[i] = nullptr;
-	for (unsigned int i = 0; i < 14; ++i)
+	for (unsigned int i = 0; i < 13; ++i)
 		m_buttons[i] = nullptr;
```
Comment (comex) [inline Source/Core/DolphinWX/TASInputDlg.cpp]: Please use `ArraySize`.

File: Source/Core/DolphinWX/TASInputDlg.h [hunk 4]
```diff
@@ -92,32 +95,54 @@ class TASInputDlg : public wxDialog
...
+
+		enum
+		{
+			CC_L_STICK_X,
+			CC_L_STICK_Y,
+			CC_R_STICK_X,
+			CC_R_STICK_Y,
+			CC_L_TRIGGER,
+			CC_R_TRIGGER,
+		};
```
Comment (comex) [inline Source/Core/DolphinWX/TASInputDlg.h]: Should put this before the member variables.

File: Source/Core/DolphinWX/TASInputDlg.cpp [hunk 5]
```diff
@@ -181,14 +197,98 @@ void TASInputDlg::CreateWiiLayout(int num)
...
+
+	HandleExtensionChange();
+	m_has_layout = true;
+}
+
+wxBoxSizer* TASInputDlg::CreateCCLayout()
+{
+	wxBoxSizer* szr = new wxBoxSizer(wxHORIZONTAL);
+
+	for(size_t i = 0; i < ArraySize(m_cc_buttons); ++i)
```
Comment (lioncash) [inline Source/Core/DolphinWX/TASInputDlg.cpp]: space between the `for` and the `(`

File: Source/Core/DolphinWX/TASInputDlg.cpp [hunk 6]
```diff
@@ -181,14 +197,98 @@ void TASInputDlg::CreateWiiLayout(int num)
...
 	m_main_szr->Add(m_ext_szr);
+	m_main_szr->Add(m_cc_szr);
+
+	HandleExtensionChange();
+	m_has_layout = true;
+}
+
+wxBoxSizer* TASInputDlg::CreateCCLayout()
+{
+	wxBoxSizer* szr = new wxBoxSizer(wxHORIZONTAL);
```
Comment (lioncash) [inline Source/Core/DolphinWX/TASInputDlg.cpp]: This should have a const after that `*`

**Comment by comex** [comment]
@dolphin-emu rebuild

**Pull request merged**